.github/workflows/design-complete.yml

```diff
@@ -0,0 +1,85 @@
+# When the "design-complete" label is added to an issue:
+# 1. Modifies the labels,
+# 2. Updates the assignees and milestone, and
+# 3. Generates a notification to the Calcite project manager(s)
+#
+# The secret is formatted like so: person1, person2, person3
+#
+# Note the script automatically adds the "@" character in to notify the project manager(s)
+name: "Design complete"
+
+on:
+  issues:
+    types: [labeled]
+
+jobs:
+  notify-designer-new-component:
+    if: github.event.label.name == 'design-complete'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/github-script@v6
+        env:
+          managers: ${{secrets.CALCITE_MANAGERS}}
+        with:
+          github-token: ${{ secrets.ADMIN_TOKEN }}
+          script: |
+            const { managers } = process.env;
+            const { label } = context.payload;
+
+            if (label && label.name === "design-complete") {
+
+              // Add a "@" character to notify the user
+              const calcite_managers = managers.split(",").map(v => " @" + v.trim());
+
+              const { data: issue } = await github.rest.issues.get({
+                issue_number: context.issue.number,
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+              });
+
+              /* Modify labels */
+
+              // Remove "Design" label
+              await github.rest.issues.removeLabel({
```

Comment:
The API calls are pretty expressive, so one suggestion to remove some comments with duplicate info is to use an object for common props and spread into each call:

```js
const commonProps = { 
  issue_number: context.issue.number,
  owner: context.repo.owner,
  repo: context.repo.repo
};

await github.rest.issues.removeLabel({
  name: "design",
  ...commonProps
});

await github.rest.issues.removeLabel({
  name: "1 - assigned",
  ...commonProps
});
```